### PR TITLE
wrudder

### PR DESCRIPTION
**Pull request by wrudder**
_If you had no problems you can just leave it empty_

> How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible
part 1  = 3
part 2 = 7

> Was there anything that you struggled with?

part 2 - got very stuck storing my function results - the below seemed to work but know its probably repeating / has useless code

> Is there any particular code you want me to look into?
wasnt confident in how to store values after a function
> Is there anything that you'd like some further information on?

> Roughly how long did it take?
part 1 - 30 mins 
part 2  - 3 hours mainly because i was finding code that put me off and got distracted with silly mistakes

### CODE REVIEW

**Comment by wrudder** [comment]
'return' better understood now from last weeks homework - having issues starting questions.. i

 tend to draw a blank when first reading the question and trying to structure the argument

for / else easily understood.

loops are  a little more tricky for me  and need to do some more practise on this.

th

File: William Rudder/homework1.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+areaOfCircle(2);
+
+//BONUS
+
+
+//Take half of the number and store the result.
+
+const number = 4;
+
+function rule () {
```
Comment (CamillaCdC) [inline William Rudder/homework1.js:47]: please use the `const rule = function () { }` syntax for declaring functions.

**Comment by CamillaCdC** [comment]
If you're having trouble starting a function, try using some pseudo code. This just means write the instructions in English-ish words. Then below each statement try writing that in code. Helps to break down the problem into a list of steps.

For example: 
QUESTION: Write a function called squareNumber that will take one argument (a number), square that number, and return the result. It should also log a string like "The result of squaring the number 3 is 9."

PSEUDO CODE: 
`
const SquareNumber = function ( number ) {
    // take number argument and square it
    const result = number * number
    // return the result of above in a string like "The result of squaring the number 3 is 9."
   return ` The result of squaring the number ${ number } is ${ result }. `
}
`

**Comment by CamillaCdC** [comment]
I'd also recommend creating a better folder structure than just having files inside your named folder. Ideally create a folder for each week and then within the week folder have a folder for each separate day's homework. will make it easier for you to keep track of it, and easier for me to review!

**Pull request closed without merging**